iocage/lib/Config/Jail/Defaults.py

```diff
@@ -109,6 +109,7 @@ class JailConfigDefaults(iocage.lib.Config.Jail.BaseConfig.BaseConfig):
         "exec_clean": 1,
         "exec_fib": 1,
         "exec_prestart": None,
+        "exec_started": None,
```

Comment:
we should probably rename this now to be in line with https://svnweb.freebsd.org/base?view=revision&revision=337863